src/main/app/idam/jwtExtractor.ts

```diff
@@ -1,12 +1,15 @@
 import * as config from 'config'
+import * as toBoolean from 'to-boolean'
+
+const oauthEnabled = toBoolean(config.get('featureToggles.idamOauth'))
 
 export default class JwtExtractor {
 
   static extract (req: any): string {
     if (!req) {
       return undefined
     }
-    if (req.query && req.query.jwt) {
+    if (!oauthEnabled && req.query && req.query.jwt) {
```

Comment:
in other places you assume `req.query` is always an object and you have: `req.query.jwt`

Reply:
no logic changed here. don't really want to spend any time here 👍